### DISCUSSION TITLE
Modulators not visible in panel

### DISCUSSION THREAD

**Discussion by Christedge** [RomanKubiak/ctrlr#456] [Q&A] [answered]
Dear all,

being new to Ctrlr development, I'm trying to create a patch viewer for a Sequential Take5 based on an existing panel. In the panel, I created several rotaries below the bottom row of DEST popups, named modAmnt-1 through modAmnt-16. They appear in the list of modulators and can be accessed in the code, but they do not appear in the UI.

Any hint how to make the visible again is much appreciated.

Thanks a bunch,

ce

![screenshot](https://user-images.githubusercontent.com/33388286/163426458-22023127-0940-4725-a6d4-1b554e3a4e20.png)
[Take5.bpanelz.zip](https://github.com/RomanKubiak/ctrlr/files/8490552/Take5.bpanelz.zip)

**Comment by damiensellier** [comment, marked as answer]
Hi, I just took a look at your panel and from what I can see, there's no image in the resources pool.
for ex : modulator-34 is linked to "bool_button" but the file is not there so the button is missing.
except from that everything seems to be alright, all the mods are on the same layer and visibility is set to true.
Can you please point me out a modulator by name that is not showing? 
thanks

EDIT: I'm checking your "modAmnt-X" mods right now and come back to you asap.

**Reply by damiensellier** [reply]
Ok I got it...
Your modAmnt-X mods are all set in a group called "modulator-7" whose position X Y and size X Y are 8 296 690 86
Your modulator  modAmnt-15 is  at 1256 800 59 54 so it's out of the group bounds and out of visibility.
When you set a modulator in a group the init position x0,y0 is the top left corner of the group position not the canvas origin.

You either have to keep the position of the modulator and remove them from the group OR change the position of the modulator so that it is going to be within the visible bounds of Grp "Modulator-7".

Component group properties:
![COMP GROUP](https://user-images.githubusercontent.com/55164385/163437814-97abbd7c-1bf5-4b25-a44f-44e8dab24eb6.png)

Component position and size properties (Pos X, Pos Y, Width, Height):
![Component position and size](https://user-images.githubusercontent.com/55164385/163437890-2fff59bf-9773-47cc-be52-35ad384afd27.png)

**Comment by Christedge** [comment]
Hi, thanks for the support. Much appreciated. I miss "modAmnt-1" through "modAmnt-16".

**Reply by damiensellier** [reply]
Just check my previous answer above for explanation
[Take 5_1_10_2022-04-14_13-05.panel.zip](https://github.com/RomanKubiak/ctrlr/files/8491013/Take.5_1_10_2022-04-14_13-05.panel.zip)
, the panel revision is attached here.

![Capture d’écran, le 2022-04-14 à 13 08 07](https://user-images.githubusercontent.com/55164385/163438977-70f9d795-6717-4203-a61e-1d9e9f9e8c20.png)

**Reply by Christedge** [reply]
Sounds plausible. I assume I screwed it up by copy pasting some existing button without paying attention to the group. Thanks a bunch for the immediate help. I'm just continuing the work with the panel you provided. Happy easter :) .

**Reply by damiensellier** [reply]
Thanks a lot, welcome to the Ctrlr community. Don't hesitate to ask any question if you need. 
If you think your problem has been addressed, please select the appropriate post and mark it as the answer.
Have a nice weekend
Damien
